new_lineage_panel/src/graph.ts

```diff
@@ -340,12 +349,12 @@ const processColumnLineage = async (
     right ? contains(curr, e.source) : contains(curr, e.target)
   );
   const newCurr = columnLineage.map((e) => (right ? e.target : e.source));
-  const collectColumns: Record<string, string[]> = {};
+  const collectColumns: Record<string, {column: string, lensType?: LensTypes}[]> = {};
```

Comment:
i think we should create separate type for collectColumns. This is getting in too many places and will probably change in future.

Reply:
done